types/github.com.hashicorp.nodeenrollment.types.v1.proto

```diff
@@ -86,6 +86,7 @@ message NodeInformation {
   bytes certificate_public_key_pkix = 2 [json_name="certificate_public_key_pkix"];
   KEYTYPE certificate_public_key_type = 3 [json_name="certificate_public_key_type"];
   repeated CertificateBundle certificate_bundles = 6 [json_name="certificate_bundles"];
+  string node_id = 7 [json_name="node_id"];
```

Comment:
Is it worth documenting how this differs from `id` above?

Reply:
Sure, I'll add that, thanks!

Reply:
Sorry, could you add it to the line just above this? That means it will make its way into the generated struct.
```suggestion
// NodeId is an external identifier for this node that does not change
  string node_id = 7 [json_name="node_id"];
```